.github/workflows/kubernetes_test.yaml

```diff
@@ -14,6 +14,7 @@ on:
   push:
     branches:
       - main
+      - release/**
```

Comment:
🤔 Since we know the format for our CalVer releases, I suggest replacing it with 

`release/v[0-9].[0-9]+'`

e.g. refer to https://github.com/Quansight/qhub/tree/release/2022.10.1

Reply:
Ahh good catch! This should be merged into `release/2022.10.1` ... I'll likely need to close this and open a new PR 😅

As for the regex, do we want to include a `v` in the branch name `release/v2020.10.1`?

Reply:
I prefer without the `v`